Workiva/ixbrl-viewer
# Add print button to document viewer

#### Description:
Added a print button to the document viewer so that if users want to print the document they can do so without printing out the inspector panels.

#### Testing:
Build the ixbrlviewer.js file locally by running `npm run prod`. Then test the following scenarios:

- **Single Instance Filing Doc**
     - Build a document with the following command: `python <path to ixbrl>ixbrl-viewer/samples/build-viewer.py --out out_file.html <path to filing doc html>`
     - Verify that the print button appears correctly in the viewer pane and that it correctly launches a print dialog that includes page breaks and formatting.
 - **Multi Instance Filing Doc**
     - Build a document with the following command: `python <path to ixbrl>ixbrl-viewer/samples/build-viewer.py --out out_dir <path to filing doc zip>`
     - Multi Instance Filing Doc can be found on the ticket or [here](https://www.sec.gov/Archives/edgar/data/78003/000007800320000014/0000078003-20-000014-xbrl.zip).
     - Verify that the print button appears correctly in the viewer pane and that it correctly launches a print dialog for each of the documents in the document tab bar. Print dialog should include page breaks and formatting.
 

**review**
@Workiva/xt 
@paulwarren-wk
